scrape.py

```diff
@@ -28,6 +30,8 @@ def get_duration(seconds):
     minutes, seconds = divmod(seconds, 60)
     return f"{minutes} mins {seconds} secs"
 
+def format_list(list):
+    return [html2text.html2text(f"{r}") for r in list.contents]
```

Comment:
Question: Is it possible to just pass in `str(list)` here? Save iterating, converting to list and back again?

Reply:
done :)